roles/openshift_serviceaccounts/tasks/main.yml

```diff
@@ -9,7 +9,8 @@
 - name: create the service account
   shell: >
        echo {{ lookup('template', '../templates/serviceaccount.j2')
-               | from_yaml | to_json | quote }} | {{ openshift.common.client_binary }}  create -f -
+               | from_yaml | to_json | quote }} | {{ openshift.common.client_binary }}
+               -n {{ openshift_serviceaccounts_namespace }} create -f -
```

Comment:
The indentation is the problem. We can bring each everything under `shell: >` to a two space indent.